guides/common/modules/proc_configuring-for-uefi-http-boot-ipv6-provisioning.adoc

```diff
@@ -0,0 +1,39 @@
+[id="configuring-for-uefi-http-boot-ipv6-provisioning_{context}"]
+= Configuring {Project} for UEFI HTTP Boot IPv6 Provisioning
+
+Use this procedure to configure {Project} to provision hosts in an IPv6 network with UEFI HTTP Boot provisioning.
+
+.Prerequisites
+
+* Ensure that your clients can access DHCP and TFTP servers.
+
+* Ensure that the UDP ports 67 and 68 are accessible by the client so that the client can send and receive a DHCP request and offer.
```

Comment:
To quote [RFC2131 section 4.1](https://tools.ietf.org/html/rfc2131#section-4.1):

>   DHCP uses UDP as its transport protocol.  DHCP messages from a client
>   to a server are sent to the 'DHCP server' port (67), and DHCP
>   messages from a server to a client are sent to the 'DHCP client' port
>   (68).

That means UDP port 67 must be open on the DHCP server in the `INPUT` chain and UDP port 68 in the `OUTPUT` chain. The current phrasing suggests you should open port 68 on the server to incoming traffic.

```suggestion
* Ensure that the UDP ports 67 and 68 are accessible by clients so clients can send DHCP requests and receive DHCP offers.
```